lib/engine/part/revenue_center.rb

```diff
@@ -19,14 +19,25 @@ def initialize(revenue, **opts)
       end
 
       # number, or something like "yellow_30|green_40|brown_50|gray_70|diesel_90"
-      def parse_revenue(revenue)
+      def parse_revenue(revenue, format)
```

Comment:
format = nil so the tests don't need to pass nil in

Reply:
done, should be ready for merge.